doc-content/enterprise-only/openshift/externaldb-build-proc.adoc

```diff
@@ -11,61 +11,79 @@ Complete the steps in this build procedure to provide drivers for any of the fol
-. Unzip the file and, using the command line, change to the `templates/contrib/jdbc` directory of the unzipped file. This directory contains the source code for the custom build.
-. Change to one of the following subdirectories:
+** For MariaDB:
 +
-** For Microsoft SQL Server, `mssql-driver-image`
-** For MariaDB, `mariadb-driver-image`
-** For IBM DB2, `db2-driver-image`
-** For Oracle Database, `oracle-driver-image`
-** For Sybase, `sybase-driver-image`
+[subs="attributes,verbatim,macros"]
+----
+make build mariadb
+----
 +
-. Enter one of the following commands:
+** For IBM DB2:
 +
---
-** For Microsoft SQL Server or MariaDB:
-
 [subs="attributes,verbatim,macros"]
 ----
-../build.sh
+make build db2
 ----
-
-** For IBM DB2, Oracle Database, or Sybase:
-
++
+** For Oracle Database:
++
 [subs="attributes,verbatim,macros"]
 ----
-../build.sh --artifact-repo=<local_path_or_url>
+make build oracle artifact=/tmp/ojdbc7.jar version=7.0
 ----
-
-Replace `<local_path_or_url>` with the path to the local directory or the URL for the HTTP server where the driver is provided. Example:
-
++
+In this command, replace `/tmp/ojdbc7.jar` with the pathname of the downloaded Oracle Database driver and `7.0` with the version of the driver.
++
+** For Sybase:
++
+[subs="attributes,verbatim,macros"]
 ----
-../build.sh --artifact-repo=/home/builder/drivers
-../build.sh --artifact-repo=http://nexus.example.com/nexus/content/groups/public
+make build sybase artifact=/tmp/jconn4-16.0_PL05.jar version=16.0_PL05
 ----
-
-If you want to configure your OpenShift docker registry address in the process, add also the `--registry=<registry_name.domain_name:port>` parameter to your build command, as illustrated in the following examples:
-
++
+In this command, replace `/tmp/jconn4-16.0_PL05.jar` with the pathname of the downloaded Sybase driver and `16.0_PL05` with the version of the driver.
++
+. Run the following command to list the Docker images that are available locally:
++
+[subs="attributes,verbatim,macros"]
 ----
-../build.sh --registry=docker-registry.custom-domain:80
-../build.sh --artifact-repo=/home/builder/drivers --registry=docker-registry.custom-domain:80
+docker images
 ----
---
++
+Note the name of the image that was built, for example, `jboss-kie-db2-extension-openshift-image`, and the version tag of the image, for example, `11.1.4.4` (not the `latest` tag).
++
+. Access the registry of your OpenShift environment directly and push the image to the registry. Depending on your user permissions, you can push the image into the `openshift` namespace or into a project namespace. For instructions about accessing the registry and pushing the images, see https://docs.openshift.com/container-platform/3.11/install_config/registry/accessing_registry.html#access[Accessing the Registry Directly].
+. When configuring your {KIE_SERVER} deployment, set the following parameters:
```

Comment:
do not forget to instruct the user to use the external-db template, which is the only one which is configured to configure the extension image.

Reply:
This configuration is copied into other templates in a template modification procedure that we have in the docs. I'll add something to this effect.